README.md

```diff
@@ -139,37 +146,44 @@ Vanilla - /whitelist on
 ## Unlock books to re-edit
 
 ## Chairs
-CraftBook  
-BetterChairs
+[CraftBook](https://www.spigotmc.org/resources/craftbook.2083/)  
+[BetterChairs](https://www.spigotmc.org/resources/betterchairs-remastered.84809/)  
+[GSit](https://www.spigotmc.org/resources/gsit-modern-sit-seat-and-chair-lay-and-crawl-plugin-1-13-x-1-17-x.62325/)
 
 ## Backpacks
 
 ## Server MOTD
 You can simply use your server.properties file to achieve this.  
-ServerListPlus
+[ServerListPlus](https://www.spigotmc.org/resources/serverlistplus.241/)  
+[PistonMOTD](https://www.spigotmc.org/resources/pistonmotd-rgb-and-1-17-support.80567/)
 
 ## Elytra cost/control
+[Purpur](https://purpur.pl3x.net/)
 
 ## Droppable Spawners
+[SilkSpawners](https://www.spigotmc.org/resources/silkspawners-%E2%98%85-ready-to-rock-1-8-1-17-supported-%E2%98%85.7811/)  
+[Purpur](https://purpur.pl3x.net/)
 
 ## Social/Command Spy
 
 ## Vanishing
-EssentialsX  
```

Comment:
Doesn't Essentials(X) handle this?

Reply:
It does, but afaik it's not packet vanish and some clients can see it

Reply:
Ahh, could always specify that in the worst-case scenario rather than remove it altogether